docs/core/windows-prerequisites.md

```diff
@@ -13,45 +13,125 @@ ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
-![Screenshot of Visual Studio 2017 installation with the ".NET Core cross-platform development" workload selected](./media/windows-prerequisites/vs_workloads.jpg)
+* Windows 7 SP1
+* Windows 8.1
+* Windows 10
+* Windows Server 2008 R2 SP1 (Full Server or Server Core)
+* Windows Server 2012 SP1 (Full Server or Server Core)
+* Windows Server 2012 R2 (Full Server or Server Core)
+* Windows Server 2016 (Full Server, Server Core, or Nano Server)
+
+See the [.NET Core Release Notes](https://github.com/dotnet/core/blob/master/release-notes/2,0/2.0.md) for the full set of supported operating systems.
+
+**.NET Core 2.0 Dependencies**
+
+.NET Core 2.0 requires the Visual C++ Redistributable when running on Windows versions earlier than Windows 10 and Windows Server 2016. This dependency is automatically installed by the .NET Core 2.0 installer.
+
+[Microsoft Visual C++ 2015 Redistributable Update 3](https://www.microsoft.com/en-us/download/details.aspx?id=52685) must be manually installed when:
+   * Installing .NET Core 2.0 with the [installer script](./tools/dotnet-install-script.md).
+   * Deploying a self-contained .NET Core 2.0 application.
+
+> [!NOTE]
+> <em>For Windows 7 and Windows Server 2008 machines only:</em><br>
+> Make sure that your Windows installation is up-to-date and includes hotfix [KB2533623](https://support.microsoft.com/help/2533623) installed through Windows Update.
+---
+
+## Developing .NET Core Applications in Visual Studio 2017
 
-There are different editions of Visual Studio 2017. You can download [Visual Studio Community 2017](https://www.visualstudio.com/downloads/) for free to get started.  To learn more about the Visual Studio installation process, see [Install Visual Studio 2017](/visualstudio/install/install-visual-studio).
+You can use any editor to develop .NET Core applications using the .NET Core SDK.  [Visual Studio 2017](#visual-studio-2017) provides an integrated development environment for .NET Core apps on Windows.
 
-To verify that you're running the latest version of Visual Studio 2017, do the following:
+There are several versions of Visual Studio 2017. You can download [Visual Studio Community 2017](https://www.visualstudio.com/downloads/) for free.
 
- * On the **Help** menu, choose **About Microsoft Visual Studio**.
- * In the **About Microsoft Visual Studio** dialog, the version number should be 15.0.26228.4 or higher.
+To learn more about the Visual Studio installation process, see [Install Visual Studio 2017](/visualstudio/install/install-visual-studio).
 
 You can read more about the changes in Visual Studio 2017 in the [release notes](https://www.visualstudio.com/news/releasenotes/vs2017-relnotes).
+
+# [.NET Core 1.x](#tab/netcore1x)
+
+> [!IMPORTANT]  It's possible to use Visual Studio 2015 for development, but it's not recommended for the following reasons:
```

Comment:
new line after [!IMPORTANT] (follow same pattern as [!NOTE] in this doc. might be the reason why the formatting is broken on this one. also wondering if we should move this one to after the tabs or maybe after the VS 2017 instructions so customers don't lose focus